dts/common/nordic/nrf54h20.dtsi

```diff
@@ -806,11 +806,10 @@
 			};
 
 			comp: comparator@983000 {
-				compatible = "nordic,nrf-comp";
+				compatible = "nordic,nrf54-comp";
```

Comment:
Such compatible does not seem to be defined.

Reply:
correct, we don't have drivers for them yet

Reply:
Added support for them, turns out they can be covered by the same driver, `nrf-comp` :)